src/stores/DataStore.js

```diff
@@ -32,7 +33,9 @@ class DataStore extends Collection {
    */
   resolve(idOrInstance) {
     if (idOrInstance instanceof this.holds) return idOrInstance;
-    if (typeof idOrInstance === 'string') return this.get(idOrInstance) || null;
+    // eslint-disable-next-line valid-typeof
+    if (typeof idOrInstance === 'bigint') return this.get(Snowflake.coerce(idOrInstance)) || null;
+    if (typeof idOrInstance === 'string') return this.get(Snowflake.coerce(idOrInstance)) || null;
```

Comment:
these checks return the same thing could you not use `typeof x === y || typeof z === a`

Reply:
I would've, but the max-len rule- so I'd have to newline it anyway, so I decided to just keep it like that.

Reply:
I'd prefer doing what Jacz said and moving the actual return statement into another line so it's not repeated

Reply:
```
if (['bigint', 'string'].includes(typeof idOrInstance)) return this.get(Snowflake.coerce(idOrInstance)) || null;
```
Because I feel like it had to be said. :omegalul:

Reply:
ew no, don't do it shay's way